integreat_cms/cms/templates/_base.html

```diff
@@ -211,12 +196,19 @@
                         {% translate "Imprint" %}
                     </a>
                 {% endif %}
-                {% if perms.cms.view_user or perms.cms.view_languagetreenode %}
+                {% if SUMM_AI_ENABLED and perms.cms.manage_translations or DEEPL_ENABLED and perms.cms.manage_translation or perms.cms.view_user or perms.cms.view_languagetreenode %}
```

Comment:
```suggestion
                {% get_mt_visibility request.region perms as show_mt_section %}
                {% if show_mt_section or perms.cms.view_user or perms.cms.view_languagetreenode %}
```
I suggest to also add a check whether summ ai is enabled in the current region and to move this complex condition into a template tag (e.g. to [base_filters.py](https://github.com/digitalfabrik/integreat-cms/blob/develop/integreat_cms/cms/templatetags/base_filters.py)):
```
@register.simple_tag
def get_mt_visibility(region, perms):
    return "cms.manage_translations" in perms and (
        settings.DEEPL_ENABLED or settings.SUMM_AI_ENABLED and region.summ_ai_enabled
    )
```

Reply:
It's a good idea 😃 Thank you for suggestion!